.bumpversion.cfg

```diff
@@ -0,0 +1,18 @@
+[bumpversion]
+current_version = 0.2.0
+commit = False
+tag = False
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<release>[a-z]+)(?P<build>\d+))?
```

Comment:
including the build number seems like a good idea, but the 'build' part here seems to always be '0'